web/components/rsvp/volunteer/index.js

```diff
@@ -138,17 +145,43 @@ class volunteerRSVP extends React.Component {
             name="dietRestrictions"
             label="Do you have any dietary restrictions or allergies?"
             placeholder="Enter an option or choose from the dropdown"
-            className="margin-top-mega margin-bottom-xxl"
+            className="margin-top-mega margin-bottom-giga"
             onChange={({ value: newDietRestriction }) => this.setState({
               dietRestriction: newDietRestriction })}
             value={dietRestriction}
             isSearchable
             allowNewOption
           />
+          <CheckboxGroup
+            name="orientationTimes"
+            label={(
+              <div> In order to volunteer for nwHacks 2019, it is required that you attend one
+                orientation session. Please check all the times you can attend:
+              </div>
+)}>
```

Comment:
I'm surprised that eslint doesn't complain about this haha, can we add a couple of spaces to this line?

Reply:
Yep!